src/CoreBundle/Bridge/Chameleon/Module/NavigationTree/NavigationTree.php

```diff
@@ -572,7 +572,12 @@ private function createRootTree(): array
             $defaultPortal = $this->portalDomainService->getDefaultPortal();
             $defaultPortalMainNodeId = $defaultPortal->fieldMainNodeTree;
 
-            $portalList = \TdbCmsPortalList::GetList();
+            $portalsInMainTreeNodeOrderQuery = '
```

Comment:
-> matchingPortalsQuery ?

The current name doesn't help me much. Probably everyone needs to look at the query itself to really understand ist.

Reply:
The query does not match anything else, it just orders the entries. Would $portalsOrderedByMainTreeNodePosition be more helpful?

Reply:
orderedPortalsQuery ?

Reply:
Renamed.